Il2CppInspector.Common/Reflection/TypeInfo.cs

```diff
@@ -16,6 +17,20 @@ public class TypeInfo : MemberInfo
         // IL2CPP-specific data
         public Il2CppTypeDefinition Definition { get; }
         public int Index { get; } = -1;
+        // This dictionary will cache all instantiated generic types out of this defintion.
```

Comment:
Typo in comment :) Also put a newline before it :)

Reply:
Will be fixed in an upcoming PR update.